pom.xml

```diff
@@ -377,6 +377,79 @@
                 </executions>
             </plugin>
         </plugins>
+        <pluginManagement>
+        	<plugins>
+        		<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+        		<plugin>
+        			<groupId>org.eclipse.m2e</groupId>
```

Comment:
Sure this mapping needs to be in this repository. Couldn't this be externalized?